remoteit/desktop
# DESK-1379 offline disconnect

### Description

Stop all web socket connections if we loose network connectivity.
Block UI if offline.
Re-establish when connectivity returns. 

### Test

1. Start connection 
2. Turn off wifi
3. Notice UI is disabled
4. Turn on wifi
5. Connection state should update to Idle
6. Reconnection should work as normal
7. Old broken connection might remain in the "OTHERS" section.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-1379>
